tests/utils/test_data.py

```diff
@@ -135,12 +146,33 @@ def test_dict_to_csv():
     """
     data = _create_data_with_values(10)
     try:
-        result1 = pipe(data, to_csv)
-        result2 = pipe(data, to_csv)
+        result1 = _pipe(data, to_csv)
+        result2 = _pipe(data, to_csv)
         filename = result1["url"]
         output = pd.read_csv(filename).to_dict(orient="records")
     finally:
         os.remove(filename)
 
     assert result1 == result2
     assert data == {"values": output}
+
+
+def test_toolz():
```

Comment:
Marked for removal: `tests.utils.test_data.test_toolz`

Dependent on resolution to comment in https://github.com/vega/altair/pull/3426/files#r1639506036